cmd/kubeadm/app/apis/kubeadm/v1beta4/doc.go

```diff
@@ -0,0 +1,271 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +k8s:defaulter-gen=TypeMeta
+// +groupName=kubeadm.k8s.io
+// +k8s:deepcopy-gen=package
+// +k8s:conversion-gen=k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm
+
+// Package v1beta4 defines the v1beta4 version of the kubeadm configuration file format.
+// This version improves on the v1beta2 format by fixing some minor issues and adding a few new fields.
+//
+// A list of changes since v1beta3:
+//   - TODO
```

Comment:
all new feature PRs must update this list.

release notes should be NONE everywhere, and we can release all features at once.

Reply:
i can add the link next to the TODO, yes